docs/source/workflows/staphylococcus_aureus_sw.csv

```diff
@@ -4,20 +4,25 @@ bracken,2.8,https://depot.galaxyproject.org/singularity/bracken:2.8--py39hc16433
 bwa,0.7.17-r1188,https://depot.galaxyproject.org/singularity/bwakit:0.7.17.dev1--hdfd78af_1
 chewbbaca,3.3.2,https://depot.galaxyproject.org/singularity/chewbbaca:3.3.2--pyhdfd78af_0
 fastqc,0.12.1,https://depot.galaxyproject.org/singularity/fastqc:0.12.1--hdfd78af_0
+flye,2.9.3,https://depot.galaxyproject.org/singularity/flye:2.9.3--py39hd65a603_0
 freebayes,1.3.6,https://depot.galaxyproject.org/singularity/freebayes:1.3.6--hbfe0e7f_2
 hostile,2.0.0,https://depot.galaxyproject.org/singularity/hostile:2.0.0--pyhdfd78af_0
 kraken2,2.1.2,https://depot.galaxyproject.org/singularity/kraken2:2.1.2--pl5321h9f5acd7_3
+medaka,2.0.1,https://depot.galaxyproject.org/singularity/medaka:2.0.1--py39hf77f13f_0
```

Comment:
Medaka has been added to the other species as well if I'm not mistaken? Could you add these there as well?